include/arch/arm/aarch64/arm_mmu.h

```diff
@@ -31,27 +31,52 @@
  * of setting PTE fields correctly.
  *
  * current usage of attrs[31:3] is:
- * attrs[3] : Access Permissions
- * attrs[4] : Memory access from secure/ns state
- * attrs[5] : Execute Permissions
+ * attrs[3] : Memory access from secure/ns state
+ * attrs[6:4] : EL0 Access / Execute permissions (RWX)
+ * attrs[9:7] : higher EL Access / Execute permissions (RWX)
```

Comment:
Based on my understanding for EL0 support, I think we can keep everything as is. We just need one extra bit here to indicate EL0 support, which will set AP[1] bit going forward.

Reply:
I see you point here, I probably got carried away with supporting all the possible combination of `{U,P}{RWX}`. I'll try to reduce the set of possible combination.